src/language/providers/closeElement.ts

```diff
@@ -16,25 +16,21 @@
  */
 
 import * as vscode from 'vscode'
-import {
-  getXsdNsPrefix,
-  insertSnippet,
-  getItemsOnLineCount,
-  checkMissingCloseTag,
-} from './utils'
+import { checkMissingCloseTag } from './closeUtils'
+import { getXsdNsPrefix, insertSnippet, getItemsOnLineCount } from './utils'
 
 export function getCloseElementProvider() {
   return vscode.languages.registerCompletionItemProvider(
     'dfdl',
     {
-      provideCompletionItems(
+      async provideCompletionItems(
```

Comment:
This was needed, along with adding `await` to line 61, for VSCode versions > 1.74.3 to avoid the auto delete of `>`.